src/controller/students.controller.ts

```diff
@@ -0,0 +1,182 @@
+/* eslint-disable prefer-const */
+/* eslint-disable import/no-unresolved */
+import express, { Request, Response } from 'express';
+import { IStudent } from '../interface/student.interface';
+import { ITacher } from '../interface/tacher.interface';
+import StudentSchema from '../models/student.model';
+import TacherSchema from '../models/tacher.model';
+import * as valdiat from './tacher.controller';
+
+const router = express.Router();
+
+// Add Student
+export const postStudent = async (req: Request, res: Response) => {
+  const student = new StudentSchema({
```

Comment:
The schema shpould be in the repository, not in the controller. same for all functions

Reply:
done